aptos-move/block-executor/src/proptest_types/types.rs

```diff
@@ -1063,224 +926,96 @@ pub(crate) enum GroupSizeOrMetadata {
     Metadata(Option<StateValueMetadata>),
 }
 
-#[derive(Debug)]
-pub(crate) struct MockOutput<K, E> {
-    pub(crate) writes: Vec<(K, ValueType)>,
-    // Key, metadata_op, inner_ops
-    pub(crate) group_writes: Vec<(K, ValueType, ResourceGroupSize, HashMap<u32, ValueType>)>,
-    pub(crate) deltas: Vec<(K, DeltaOp)>,
-    pub(crate) events: Vec<E>,
-    pub(crate) read_results: Vec<Option<Vec<u8>>>,
-    pub(crate) read_group_size_or_metadata: Vec<(K, GroupSizeOrMetadata)>,
-    pub(crate) materialized_delta_writes: OnceCell<Vec<(K, WriteOp)>>,
-    pub(crate) total_gas: u64,
-    pub(crate) skipped: bool,
+// Utility function to convert a key to a module ID
```

Comment:
supernit: the comment should specify that this is not conversion, but rather we use hash to set the address and then use "test" all the time?

Reply:
changed the name to key_to_mock_module_id, also updated the comment.